src/payments/dtos/payment-report.dto.ts

```
@@ -0,0 +1,51 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class PaymentReportItemDto {
+  @ApiProperty({ type: 'string', description: 'First Name of the user', nullable: true })
```

ESLint

[error] Replace `·type:·'string',·description:·'First·Name·of·the·user',·nullable:·true` with `⏎····type:·"string",⏎····description:·"First·Name·of·the·user",⏎····nullable:·true,⏎·`

(prettier/prettier)